docs/source/noise/row_noise.rst

```diff
@@ -7,13 +7,47 @@ Row-based Noise
 Row-based noise operates on one row of data at a time, for example by omitting
 or duplicating entire rows.
 
+Do not respond
+--------------
+
+Sometimes people don't respond to a census or survey questionnaire and are
+unable to be reached by other means such as telephone or personal visit. For the
+Decennial Census and household surveys (ACS and CPS), people are found to
+respond at different rates depending on demographics (e.g., age, sex,
+race/ethnicity) and mode of contact (e.g., mail/online, telephone, personal
+visit). To simulate nonresponse bias in the Decennial Census and household
```

Comment:
Is there any purpose to including "mode of contact" here? This was sort of part of a planned feature that we never ended up using. I think it would completely cover what we ended up including to say:

We assumed nonresponse in the Decennial Census was equal to the _net_ undercount (for the relevant demographic group), ignoring duplication. [cite PES estimates] We assumed nonresponse in the ACS was the same as in the Decennial Census, since these are conducted similarly. We assumed that CPS had the same nonresponse pattern as the Decennial Census, but with a constant 27.6% added to the nonresponse rates, since that was the nonresponse rate for CPS in July 2022. [cite BLS]

Reply:
I didn't know the details of what was implemented, I was just trying to describe it based on the concept model. I'll edit to use your description above.

Reply:
@zmbc Does that mean [this table](https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_census_synthdata/concept_model.html#id115) does **not** accurately describe what's happening in the sim?

And are you saying we did not use [the Jackson reference](https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_census_synthdata/concept_model.html#household-surveys:~:text=This%20data%20is,for%20white%20people) at all?

Reply:
Yeah, this is very confusing and I'd love to clean it up in the concept model someday. In the concept model we have documentation about "surveys in general," and then documentation about the two specific surveys. The table you linked is part of the former, and describes how to deal with different response modes.

However, for the two surveys we actually included:

* ACS has all response modes, so the "Probability of Non-Response (%)" column is always 0%
* CPS has contact modes in a different configuration, so we make an exception: "As this does not fit into the framework above, we will use the values for mail/online, telephone, and personal visits [which is all response modes, like ACS] and then apply an overall non-response rate of 27.6%."

So yes, we end up not doing anything that is based on the Jackson reference.

Reply:
Ok, thanks for the clarification. Yeah, that was not clear to me as I was reading it. I guess I should make a backlog ticket to make this clearer in the concept model...